source/common/config/utility.cc

```diff
@@ -63,20 +70,23 @@ void Utility::translateEdsConfig(const Json::Object& json_config,
                                  envoy::api::v2::ConfigSource& eds_config) {
   translateApiConfigSource(json_config.getObject("cluster")->getString("name"),
                            json_config.getInteger("refresh_delay_ms", 30000),
+                           json_config.getString("api_type", "REST_LEGACY"),
```

Comment:
Make "REST_LEGACY" a const somewhere. And also yes please make the above some static utility function w/ tests.